src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -135,12 +135,30 @@ s16 D_8082AB2C[] = {
     24, 72, 13, 22, 19, 20, 19, 27, 14, 26, 22, 21, 49, 32, 45, 60, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 16, 32, 8,
 };
 
-static u8 D_8082AB6C[][5] = {
+static u8 gPageSwitchNextButtonStatus[][5] = {
+    // PAUSE_ITEM  + SWITCH_PAGE_LEFT_PT
+    //
+    //  -> PAUSE_EQUIP
     { BTN_ENABLED, BTN_DISABLED, BTN_DISABLED, BTN_DISABLED, BTN_ENABLED },
+    // PAUSE_MAP   + SWITCH_PAGE_LEFT_PT
+    //
+    //  -> PAUSE_ITEM
     { BTN_ENABLED, BTN_ENABLED, BTN_ENABLED, BTN_ENABLED, BTN_DISABLED },
```

Comment:
this is definitely just my lack of understanding of how this is working in general, but these comments are confusing to me. I guess there are entries for each page when coming from another page? may be good to explain that above the array but idk